### PR TITLE
[bitnami/etcd] Fix scaling etcd nodes

### PR DESCRIPTION
**Pull request by tompizmor**
Fixes https://github.com/bitnami/charts/issues/791.

The main issue was loading the env-vars for the new cluster state after add a new node.

### CODE REVIEW

File: bitnami/etcd/templates/statefulset.yaml [hunk 1]
```diff
@@ -164,11 +166,11 @@ spec:
         - name: ETCD_DATA_DIR
           value: {{ template "etcd.dataDir" . }}
         - name: ETCD_ADVERTISE_CLIENT_URLS
-          value: "{{ $etcdClientProtocol }}://$(MY_POD_NAME):{{ .Values.service.port }}"
+          value: "{{ $etcdClientProtocol }}://$(MY_POD_NAME).{{ $etcdHeadlessServiceName }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.service.port }}"
```
Comment (juan131) [inline bitnami/etcd/templates/statefulset.yaml]: Why do you still need `$(MY_POD_NAME)`?

Comment (javsalgar) [reply bitnami/etcd/templates/statefulset.yaml]: I suppose because it is using headless services

Comment (tompizmor) [reply bitnami/etcd/templates/statefulset.yaml]: because it is different for each replica

etcd-0, etcd-1, etcd-2,...

Comment (juan131) [reply bitnami/etcd/templates/statefulset.yaml]: 👍

**Pull request merged**